### ISSUE TITLE
Bootstrap throws Uncaught Error: Bootstrap's JavaScript requires jQuery

### ISSUE THREAD

**Issue by anyaevostinar** [devosoft/empirical-prefab-demo#4]
The scripts to include seem to be out of date, leading to a error when running the ConfigPanel example `Uncaught Error: Bootstrap's JavaScript requires jQuery` which is fixed (via [this SO](https://stackoverflow.com/questions/22658015/bootstrap-throws-uncaught-error-bootstraps-javascript-requires-jquery)) by adding `<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.0/jquery.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">` to the beginning of the scripts.

**Comment by mmore500** [comment]
Good catch! Fixed.

Backstory: JQuery is a base Empirical dependency, so it wasn't listed here. But most people on this page will be setting up new project instead of migrating/augmenting existing Empirical project, so we should list JQuery dependnecy.

Looks like the version of Jquery that has been floating around with Empirical since the beginning of time is 1.x, so I went ahead and pinned the new script tag to a 1.x branch. 

Also took the opportunity to future proof by hard pinning ALL dependencies and adding integrity checks
